fansi/shared/src/main/scala/fansi/Fansi.scala

```diff
@@ -259,7 +259,7 @@ object Str{
     * The `0000 0000 0000 0000` int corresponds to plain text with no decoration
     *
     */
-  type State = Int
+  type State = Long
```

Comment:
Let's update the scaladoc-comment above this to reflect the new state of affairs, with `...`s to collapse the ranges